compass/ocean/tests/soma/namelist.forward

```diff
@@ -0,0 +1,27 @@
+config_ocean_run_mode = 'forward'
+config_run_duration = '0000_00:01:30'
+config_use_mom_del4 = .true.
+config_mom_del4_div_factor = 10.0
+config_use_cvmix = .true.
+config_use_cvmix_background = .true.
+config_use_cvmix_convection = .true.
+config_cvmix_background_diffusion = 5.0e-6
+config_cvmix_convective_basedOnBVF = .true.
+config_use_cvmix_shear = .true.
+config_cvmix_shear_mixing_scheme = 'KPP'
+config_use_cvmix_kpp = .true.
+config_use_bulk_wind_stress = .true.
+config_density0 = 1014.65
+config_eos_type = 'jm'
+config_eos_linear_alpha = 0.255
+config_eos_linear_beta = 0.764
+config_eos_linear_Tref = 19.0
+config_eos_linear_Sref = 35.0
```

Comment:
It's odd that we're using JM but setting parameters for the linear EOS.  Also, the linear `alpha` here doesn't match the one used in the IC.